viewer/packages/pointclouds/src/potree-three-loader/Potree.test.ts

```diff
@@ -23,4 +23,26 @@ describe(Potree.name, () => {
       expect(potreeInstance.pointBudget).toEqual(budget);
     }
   });
+
+  test('Max nodes loading is set from constant', () => {
+    const potreeInstance = new Potree(mockModelDataProvider, mockMaterialManager);
+
+    expect(potreeInstance.maxNumNodesLoading).toBeGreaterThan(0);
+    expect(potreeInstance.maxNumNodesLoading).toBeLessThanOrEqual(20);
```

Comment:
Should we just compare this directly to the constant, instead of the range (0,20]?